packages/@react-aria/numberfield/src/useNumberField.ts

```diff
@@ -88,13 +89,15 @@ export function useNumberField(props: AriaNumberFieldProps, state: NumberFieldSt
 
   let inputId = useId(id);
 
+  let {focusableProps} = useFocusable(props, inputRef);
```

Comment:
Perhaps I'm overthinking this, but what about passing `props.onBlur/onFocus/onFocusChange/keyUp/keyDown` to `useTextField` instead? `useTextField` is called further below and that hook calls `useFocusable` for us already, might be easier to add the missing events instead. In fact, would doing a `...props` in the `useTextField` call work for us (would have to override defaultValue so that it is undefined probably)? Or was that purposely omitted? 

@snowystinger ^

Reply:
if you're asking why not everything is just passed to useTextField, i think it was because of a TS issue, but we can A. Definitely try it again, that API changed a while back and it may now work, or B. just pass the list you gave in addition, it think that's perfectly reasonable and saves us some duplicate code

Reply:
Did some playing around and it seems like doing the following works (similar to useSearchField):
```
  let {labelProps, inputProps: textFieldProps} = useTextField({
    ...props,
    defaultValue: undefined,
....
```
I'm fine w/ option B if we think the `defaultValue: undefined` is a bit gross haha, but spreading the props will be more effective at supporting all the extra props/callbacks that a input field is allowed